<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NotepadClass</class>
 <widget class="QMainWindow" name="NotepadClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>781</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Notepad</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTextEdit" name="textEdit">
        <property name="placeholderText">
         <string>Code......</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser">
        <property name="placeholderText">
         <string>Waiting......</string>
        </property>
        <property name="searchPaths">
         <stringlist notr="true"/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>FontSize</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="size_edit">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>30</number>
            </property>
            <property name="value">
             <number>12</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>ImportCode</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>FontSize</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="size_browser">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>30</number>
            </property>
            <property name="value">
             <number>12</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QPushButton" name="pushButton_4">
            <property name="text">
             <string>coe</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string>bin</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>25</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>go-&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="mode_lable">
        <property name="text">
         <string>mode:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="mode">
        <property name="text">
         <string>assemble</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="change_mode">
        <property name="text">
         <string>change</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionassembler"/>
    <addaction name="actiondissembler"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionopen"/>
    <addaction name="separator"/>
    <addaction name="actionsave"/>
    <addaction name="actionsave_as"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionassembler">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/Notepad/Notepad.ico</normaloff>:/Notepad/Notepad.ico</iconset>
   </property>
   <property name="text">
    <string>assembler</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+A</string>
   </property>
  </action>
  <action name="actiondissembler">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/Notepad/Notepad.ico</normaloff>:/Notepad/Notepad.ico</iconset>
   </property>
   <property name="text">
    <string>dissembler</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionsave">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/Notepad/Notepad.ico</normaloff>:/Notepad/Notepad.ico</iconset>
   </property>
   <property name="text">
    <string>save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionsave_as">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/Notepad/Notepad.ico</normaloff>:/Notepad/Notepad.ico</iconset>
   </property>
   <property name="text">
    <string>save as</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionopen">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/Notepad/Notepad.ico</normaloff>:/Notepad/Notepad.ico</iconset>
   </property>
   <property name="text">
    <string>open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="notepad.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>NotepadClass</receiver>
   <slot>compile_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>103</x>
     <y>736</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>389</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionsave</sender>
   <signal>triggered()</signal>
   <receiver>NotepadClass</receiver>
   <slot>save_called()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionsave_as</sender>
   <signal>triggered()</signal>
   <receiver>NotepadClass</receiver>
   <slot>saveas_called()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionopen</sender>
   <signal>triggered()</signal>
   <receiver>NotepadClass</receiver>
   <slot>open_called()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionassembler</sender>
   <signal>triggered()</signal>
   <receiver>NotepadClass</receiver>
   <slot>assembler_called()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actiondissembler</sender>
   <signal>triggered()</signal>
   <receiver>NotepadClass</receiver>
   <slot>dissembler_called()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>change_mode</sender>
   <signal>clicked()</signal>
   <receiver>NotepadClass</receiver>
   <slot>switch_called()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>921</x>
     <y>736</y>
    </hint>
    <hint type="destinationlabel">
     <x>519</x>
     <y>390</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>size_edit</sender>
   <signal>valueChanged(int)</signal>
   <receiver>NotepadClass</receiver>
   <slot>change_fontsize_edit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>434</x>
     <y>635</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>435</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>size_browser</sender>
   <signal>valueChanged(int)</signal>
   <receiver>NotepadClass</receiver>
   <slot>change_fontsize_browser()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>894</x>
     <y>634</y>
    </hint>
    <hint type="destinationlabel">
     <x>452</x>
     <y>430</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>NotepadClass</receiver>
   <slot>open_called()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>418</x>
     <y>666</y>
    </hint>
    <hint type="destinationlabel">
     <x>405</x>
     <y>506</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>NotepadClass</receiver>
   <slot>export_bin_called()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>905</x>
     <y>665</y>
    </hint>
    <hint type="destinationlabel">
     <x>817</x>
     <y>519</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>NotepadClass</receiver>
   <slot>export_coe_called()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>678</x>
     <y>665</y>
    </hint>
    <hint type="destinationlabel">
     <x>615</x>
     <y>513</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>compile_clicked()</slot>
  <slot>save_called()</slot>
  <slot>saveas_called()</slot>
  <slot>open_called()</slot>
  <slot>assembler_called()</slot>
  <slot>dissembler_called()</slot>
  <slot>switch_called()</slot>
  <slot>change_fontsize_edit()</slot>
  <slot>change_fontsize_browser()</slot>
  <slot>export_coe_called()</slot>
  <slot>export_bin_called()</slot>
 </slots>
</ui>
